revscoring/features/parent_revision.py

```diff
@@ -0,0 +1,118 @@
+from . import modifiers
+from ..datasources import parent_revision, revision
+from ..languages import is_badword, is_misspelled
+from .feature import Feature
+from .revision import bytes as revision_bytes
+from .util import MARKUP_RE, NUMERIC_RE, SYMBOLIC_RE
+
+
+################################## Bytes #######################################
+
+def process_bytes(parent_revision_metadata):
+    return parent_revision_metadata.bytes \
+           if parent_revision_metadata is not None else 0
+
+bytes = Feature("parent_revision.bytes", process_bytes,
+                returns=int, depends_on=[parent_revision.metadata])
+
+def process_was_same_user(parent_revision_metadata, revision_metadata):
```

Comment:
This is another place where the property sounds more natural if associated to the pair of revisions instead of a single (the parent) revision. In the end it is using the metadata from both revisions...
(not a blocker though)